DMCompiler/DMCompiler.cs

```diff
@@ -128,9 +128,13 @@ public void AddResourceDirectory(string dir) {
                 preproc.IncludeFile(includeDir, fileName, false);
             }
 
+            // Adds the root of the DM project to FILE_DIR
+            compiler.AddResourceDirectory(Path.GetDirectoryName(files[0]));
```

Comment:
`GetDirectoryName()` could return null if the path is in the root directory. Null coalesce it to `"/"`.
```suggestion
            compiler.AddResourceDirectory(Path.GetDirectoryName(files[0]) ?? "/");
```

Reply:
Thanks I'll read the function reference next time ^^'